main.py

```
@@ -248,6 +251,12 @@ def create_stat_widget(title: str, image_path: str, value: str) -> QWidget:
         return final_widget
 
 
+    def toggle_file_tree(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `toggle_file_tree`

Add return type annotation: `None`

(ANN201)